test/functional/tag_controller_test.rb

```diff
@@ -48,6 +48,16 @@ def setup
     assert_response :success
   end
 
+  test 'only admins can added the locked tag' do
```

Comment:
Thanks for creating the test @Manasa2850.. this looks good, the `added` is in past tense, can it be in the present like `'only admins can add the locked tag'`

Also, I think another test that confirms an admin and a moderator can add a locked tag will help.

Reply:
Oh sorry my bad :sweat_smile: 
Sure, I'll add the other test too! Thanks!